### PR TITLE
Create modules folder

### PR DESCRIPTION
**Pull request by polubis**

### CODE REVIEW

File: src/core/auth/Service.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { Auth } from '.';
+
+export const logIn = (payload: Auth.Credentials) =>
+  core.post<Auth.User>('Authorization/SignIn', payload);
+
+export const logInViaGithub = () => {
+  window.location.href = `${CORE_API_PATH}GithubAuthorization/SignIn?redirectUrl=${window.location.origin}/app`;
+};
+
+export const logOut = () => core.post<null>('Authorization/SignOut');
```
Comment (marcinreszko) [inline src/core/auth/Service.tsx:14]: It has to be .tsx if there is no jsx?

File: src/modules/ModulesRouter.tsx [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+const ModulesRouter = () => {
+  return (
+    <Switch>
+      <Guard.UnprotectedRoute exact path="/login" redirect="/app" component={Login} />
+      <Guard.UnprotectedRoute exact path="/register" redirect="/app" component={Register} />
+      <Route path="/app" component={Main} />
+      <Route exact path="/" component={Home} />
+      <Route path="**" render={() => <div>Not Found Page </div>} />
+    </Switch>
+  );
```
Comment (marcinreszko) [inline src/modules/ModulesRouter.tsx:22]: Shouldn't we declare other routes as unprotected aswell? Just to keep everything consistent

File: src/modules/template-documentation/TemplateDocumentation.scss [hunk 3]
```diff
@@ -1,7 +1,7 @@
 @import 'styles/palette';
 @import 'styles/mixins';
```
Comment (marcinreszko) [inline src/modules/template-documentation/TemplateDocumentation.scss:3]: One of those can be deleted if I remember correctly

File: src/modules/ModulesRouter.tsx (hunk 2 above)
Comment (polubis) [reply src/modules/ModulesRouter.tsx:22]: Unprotected refers to - redirect somewhere if user is authorized - so authorized user cannot use this resources.

Protected refers to - redirect user to somewhere if he is not authorized - so he can access this resources only when he is auth,

Normal Route refers to - i dont care both authorized and not authorized users can access this resources - so this is ok

File: src/modules/template-documentation/TemplateDocumentation.scss (hunk 3 above)
Comment (polubis) [reply src/modules/template-documentation/TemplateDocumentation.scss:3]: Done

File: src/core/auth/Service.tsx (hunk 1 above)
Comment (polubis) [reply src/core/auth/Service.tsx:14]: Done

File: src/modules/ModulesRouter.tsx (hunk 2 above)
Comment (marcinreszko) [reply src/modules/ModulesRouter.tsx:22]: Ah ok, sure. Got it.

**Pull request merged**